src/jit/compiler.h

```diff
@@ -7092,6 +7338,20 @@ class Compiler
         return codeGen->getEmitter();
     }
 
+    // Starts or ends a "VariableLiveRange" for "varDsc" if it is borning or dying respectively.
+    // Ranges are close-open [) so nothing is done in case of being borning and dying at the same line
+    // due to be an empty range.
+    void startOrCloseVariableLiveRange(const LclVarDsc* varDsc, bool isBorning, bool isDying);
```

Comment:
Customarily, members of `Compiler` that are associated with a particular phase or functionality are given a prefix (e.g. `fg` for flowgraph-related members). I'd suggest 'si' for these, though I believe that you're thinking about refactoring some of these out of `Compiler`, which would be good. These methods could easily be misinterpreted as being related to dataflow in the JIT.